docs/Bigtable-batchsource.md

```diff
@@ -55,3 +55,24 @@ Only columns defined in schema will be included into output record.
 **onError**: Strategy used to handle errors during transformation of a text entry to record. Possible values are:
 - **Skip error** - Ignores erroneous records.
 - **Fail Pipeline** - Fails pipeline due to erroneous record.
+
+Data Type Mappings from BigQuery to CDAP
+----------
+The following table lists out different BigQuery data types, as well as the 
+corresponding CDAP data type for each BigQuery type.
+
+| BigQuery type | CDAP type   | Precision / Range           |
```

Comment:
this 3rd column is more confusing than helpful. You can just put the information in the CDAP type column. For example, timestamp (microseconds)